backend/migrations/20201107134701_remove_firstName_lastName_from_students.js

```diff
@@ -0,0 +1,17 @@
+const { tableStudents } = require('../src/models/student')
+
+exports.up = function(knex, Promise) {
+    return knex.schema.table(tableStudents, (table) => {
+        table.dropUnique(['PhoneNumber', 'FirstName', 'LastName']);
```

Comment:
In the previous migration, you made `PhoneNumber` unique, here you are removing that constraint. I think you can just do `dropUnique` in the first migration?

Reply:
In the previous migration, i made the combination of PhoneNumber + FullName unique
`table.unique(['PhoneNumber', 'FullName']);`

In this migration, it is the unique constraint for the combination of PhoneNumber + FirstName + LastName that we are dropping because we are removing the fields FirstName & LastName.

Reply:
Ah ok, didn't know that the uniqueness was across the combination. Thought it was individual.